tests/test_config_consistency.py

```diff
@@ -605,12 +614,21 @@ def setup_main(self):
     def test_main(self):
         assert self.req.status_code == 200
 
-        runtime_metrics = [
+        runtime_metrics_gauges = [
```

Comment:
Duplicate code here seems like it should be a helper function like `wait_for_runtime_metrics`

Reply:
Done in c2d226c0d12c1ca2ad95fd432f79e7c683c74961